### PR TITLE
fix: Paragraph Component wraps any content with a P tag 

### LINKED ISSUE #11350: Paragraph Component wraps any content with a P tag
**Describe the bug**
The paragraph component is used to render the richtext added via Smart Edit. As this richtext can contain various html tags depending on the CKEditor config, it should use a `div` to render its content. However, currently it uses a `p` tag to render and therefore generates semantically wrong content:

```
<cx-paragraph>
   <p>
      <h1>My H1 Heading</h1>
      <p>My Paragraph</p>
   </p><!---->
</cx-paragraph>
```

**Tell us the version of Spartacus**
- Library version: 3.1.0

**To Reproduce**
1. Go to the Page https://spartacus-demo.eastus.cloudapp.azure.com/electronics-spa/en/USD/faq
2. Inspect the Markup of the Heading "Delivery information"
3. See the wrongly nested markup structure

**Expected behavior**
The paragraph component uses a div as surrounding wrapper element

**Screenshots**
image

**Desktop (please complete the following information):**
 - OS: All
 - Browser All
 - Version All

**Smartphone (please complete the following information):**
 - Device: All
 - OS: All
 - Browser All
 - Version All

### PR DESCRIPTION
**Pull request by bit172**
close #11350

### CODE REVIEW

File: projects/storefrontlib/src/cms-components/content/paragraph/paragraph.component.html [hunk 1]
```diff
@@ -1 +1 @@
-<p *ngIf="component.data$ | async as data" [innerHTML]="data.content"></p>
+<div *ngIf="component.data$ | async as data" [innerHTML]="data.content"></div>
```
Comment (Platonn) [inline projects/storefrontlib/src/cms-components/content/paragraph/paragraph.component.html]: It's a breaking change in the DOM (and for any customer styling) so should be documented in our 4_0.md migration docs.

See https://sap.github.io/spartacus-docs/breaking-changes/#avoiding-breaking-changes

Comment (bit172) [reply projects/storefrontlib/src/cms-components/content/paragraph/paragraph.component.html]: Good to know! I will add it to the migration docs.

**Comment by Platonn** [comment]
Somehow we missed the 4.0 release window. Due to breaking change, wow we need to postpone it to 5.0 and document the breaking change in the 5_0.md, not 4_0.md.

File: docs/migration/5_0.md [hunk 2]
```diff
@@ -190,6 +190,10 @@ configurations on every navigation that is not configurator related, and deletes
 
 - `common.breadcrumbs` was added to `common.ts`
 
+### ParagraphComponent
+
+- Replaced the HTML `p` tag with a `div`.
```
Comment (dydome) [inline docs/migration/5_0.md]:
```suggestion
- `p` has been replaced by `div` tag.
```

**Review by dydome — CHANGES_REQUESTED** [review]

**Review by dydome — APPROVED** [review]

**Pull request merged**